src/Common/Logging/EventAuditLogger.php

```diff
@@ -632,6 +640,27 @@ public function auditSQLEvent($statement, $outcome, $binds = null)
             }
         }
 
+        /**
+         * Avoid logging sensitive information in logs.
+         * For any insert operation, if the current table contains any field containing sensitive information,
+         * the value for that field is masked to match its length.
+         */
+        foreach (self::SENSITIVE_RECORDS as $table => $fields) {
```

Comment:
would put the querytype == insert prior to the foreach (no reason to loop through the sensitive records array if the querytype does not equal insert)

Reply:
@bradymiller I believe this is now resolved? Good for us to bring this in?